src/plugins/input_control_vis/public/input_control_vis_type.ts

```diff
@@ -21,14 +21,14 @@ export function createInputControlVisTypeDefinition(
   return {
     name: 'input_control_vis',
     title: i18n.translate('inputControl.register.controlsTitle', {
-      defaultMessage: 'Controls',
+      defaultMessage: 'Input controls',
```

Comment:
We chose to re-name these to help distinguish between the **old** visualization controls (now *input controls*) and the **new** dashboard controls (just *controls*).